tools/walletextension/accountmanager/account_manager.go

```diff
@@ -77,17 +79,16 @@ const emptyFilterCriteria = "[]" // This is the value that gets passed for an em
 
 // determine the client based on the topics
 // if none is found use the first client and assume this is a lifecycle method
-func (m *AccountManager) suggestSubscriptionClient(rpcReq *RPCRequest) (rpc.Client, error) {
-	var client rpc.Client //= m.unauthedClient // todo (@ziga) - support lifecycle events through an unauthed client
+func (m *AccountManager) suggestSubscriptionClient(rpcReq *RPCRequest) ([]rpc.Client, error) {
+	clients := make([]rpc.Client, 0, len(m.accountClients))
 
-	// by default use the first client for lifecycle events
+	// if none is found subscribe to all accounts
```

Comment:
this comment is not very clear.
I would say: "by default, if no client is identified as a candidate, then subscribe to all accounts"

Reply:
fixed